src/Maybe.gren

```diff
@@ -47,21 +47,55 @@ type Maybe a
     | Nothing
 
 
-{-| Provide a default value, turning an optional value into a normal
-value. This comes in handy when paired with functions like
-[`Dict.get`](Dict#get) which gives back a `Maybe`.
+{-| Checks to see if the [Maybe](#Maybe) is `Just`, and that the contained value
+equals a provided constant.
+
+    hasValue 5 (Just 5) == True
+
+    hasValue 5 (Just 3) == False
+
+    hasValue 5 Nothing == False
+
+-}
+hasValue : a -> Maybe a -> Bool
+hasValue value maybe =
```

Comment:
Can you not do `== Just a` in Gren? Or is this meant to be faster?

Reply:
Faster (no allocation, simpler equality check).